integration-tests/integration_test.go

```diff
@@ -427,7 +427,8 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	err = s.collector.Launch()
 	s.Require().NoError(err)
 
-	scheduled_curls_image := qaImage("quay.io/rhacs-eng/qa", "collector-schedule-curls");
+	image := "quay.io/rhacs-eng/qa"
```

Comment:
The cumulative changes in this file also end up being just cosmetic; drop them entirely.